CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * Added Ansys installation path for linux to Additive object initializer. [#308](https://github.com/ansys/pyadditive/issues/308)
 * CSV files with empty thermal parameters of a microstructure simulation will be imported [#309](https://github.com/ansys/pyadditive/issues/309)
 * Gracefully return from run_simulations when no simulations meet criteria [#318](https://github.com/ansys/pyadditive/pull/318)
+* Parametric Studies created by importing CSV files will have unique row indices [#364] (https://github.com/ansys/pyadditive/issues/364)
```

Comment:
This doesn't really describe what was fixed. Maybe say something like, "Fixed simulation results being updated for incorrect simulations in parametric studies."